.jenkins/Packaging.Jenkinsfile

```diff
@@ -48,11 +52,23 @@ def WindowsPackaging(String build_type) {
     }
 }
 
-parallel "1604 SGX1FLC Package Debug" :          { LinuxPackaging('1604', 'Debug') },
+try{
+    oe.emailJobStatus('STARTED')
+
+    parallel "1604 SGX1FLC Package Debug" :          { LinuxPackaging('1604', 'Debug') },
          "1604 SGX1FLC Package Release" :        { LinuxPackaging('1604', 'Release') },
          "1604 SGX1FLC Package RelWithDebInfo" : { LinuxPackaging('1604', 'RelWithDebInfo') },
          "1804 SGX1FLC Package Debug" :          { LinuxPackaging('1804', 'Debug') },
          "1804 SGX1FLC Package Release" :        { LinuxPackaging('1804', 'Release') },
          "1804 SGX1FLC Package RelWithDebInfo" : { LinuxPackaging('1804', 'RelWithDebInfo') },
          "Windows Debug" :                       { WindowsPackaging('DEBUG') },
          "Windows Release" :                     { WindowsPackaging('RELEASE') }
+}catch(Exception e)
+{
```

Comment:
I think we are doing the other bracket format, i.e. 

`}catch(Exception e) {
 `

Reply:
Thanks, should be done.